src/anomalib/models/efficientad/torch_model.py

```diff
@@ -123,7 +123,6 @@ class Decoder(nn.Module):
     def __init__(self, out_channels, padding, img_size, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
         self.img_size = img_size
-        self.last_upsample = 64 if padding else 56
```

Comment:
Was this commited intentionally?

Reply:
yes this line is not used as the same variable is (correctly) defined one line after..